components/measurement/nettests/RiseupVPN.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types'
+
+const RiseupVPNDetails = ({ measurement, render }) => {
+  // TODO(bassosimone,cyberta): possibly refine the algorithm
```

Comment:
@cyberta: does this look good to you as a first stab at scoring the measurement?